usaspending_api/api_contracts/contracts/v2/search/spending_by_category/federal_account.md

```diff
@@ -0,0 +1,189 @@
+                    "previous": null,
+                    "hasNext": false,
+                    "hasPrevious": false
+                },
+                "results": [
+                    {
+                        "amount": 4575113.34,
+                        "id": 3573,
+                        "code": "012-4336",
+                        "name": "Commodity Credit Corporation Fund, Agriculture"
+                    },
+                    {
+                        "amount": 548022.0,
+                        "id": 5184,
+                        "code": "070-0411",
+                        "name": "Federal Assistance, Domestic Nuclear Detection Office, Homeland Security"
+                    }
+                ],
+                "messages": [
+                    "For searches, time period start and end dates are currently limited to an earliest date of 2007-10-01.  For data going back to 2000-10-01, use either the Custom Award Download feature on the website or one of our download or bulk_download API endpoints as listed on https://api.usaspending.gov/docs/endpoints."
+                ]
+            }
+
+# Data Structures
+
+## CategoryResult (object)
++ `id` (required, number)
+    The id is the database key.
++ `name` (required, string, nullable)
++ `code` (required, string, nullable)
+    `code` is a user-displayable code (such as a program activity or NAICS code, but **not** a database ID). When no such code is relevant, return a `null`.
++ `amount` (required, number)
+
+## PageMetadataObject (object)
++ `page` (required, number)
++ `hasNext` (required, boolean)
+
+## Filter Objects
+### FilterObject (object)
++ `keywords` : `transport` (optional, array[string])
++ `time_period` (optional, array[TimePeriodObject], fixed-type)
++ `place_of_performance_scope` (optional, enum[string])
+    + Members
+        + `domestic`
+        + `foreign`
++ `place_of_performance_locations` (optional, array[LocationObject], fixed-type)
++ `agencies` (optional, array[AgencyObject], fixed-type)
++ `recipient_search_text`: `Hampton` (optional, array[string])
++ `recipient_id` (optional, string)
+    A unique identifier for the recipient which includes the recipient hash and level.
++ `recipient_scope` (optional, enum[string])
+    + Members
+        + `domestic`
+        + `foreign`
++ `recipient_locations` (optional, array[LocationObject], fixed-type)
++ `recipient_type_names`: `category_business` (optional, array[string])
++ `award_type_codes` (optional, FilterObjectAwardTypes)
++ `award_ids`: `SPE30018FLGFZ`, `SPE30018FLJFN` (optional, array[string])
+    Award IDs surrounded by double quotes (e.g. `"SPE30018FLJFN"`) will perform exact matches as opposed to the default, fuzzier full text matches.  Useful for Award IDs that contain spaces or other word delimiters.
++ `award_amounts` (optional, array[AwardAmounts], fixed-type)
++ `program_numbers`: `10.331` (optional, array[string])
++ `naics_codes`: `311812` (optional, array[string])
```

Comment:
The `naics_codes` filter was updated recently to an object. It was pointed out to me in a different PR and was changed in the other spending_by_category API contracts.